libvast/vast/table_slice_encoding.hpp

```diff
@@ -24,7 +26,25 @@ enum class table_slice_encoding : uint8_t {
   msgpack, ///< The table slice is encoded using the MessagePack format.
 };
 
-/// @relates table_slice_encoding
-std::string to_string(table_slice_encoding encoding) noexcept;
-
 } // namespace vast
+
+namespace fmt {
+
+template <>
+struct formatter<vast::table_slice_encoding> : formatter<std::string_view> {
+  template <typename FormatContext>
+  auto format(const vast::table_slice_encoding& x, FormatContext& ctx) {
+    using super = formatter<std::string_view>;
```

Comment:
```suggestion
    using f = formatter<std::string_view>;
```

I'm only reluctant to use `super` as identifier here because we almost always use it as type alias for the respective base class. Since there's no inheritance here, something simpler (like `f`) might cause less confusion.

Reply:
It _is_ the base class, hence the name `super`